upstash/manifest.json

```diff
@@ -0,0 +1,57 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "b6204e82-0732-4ab4-bca6-ed2e727ea581",
+  "app_id": "upstash",
+  "display_on_public_website": false,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Publish your Upstash Redis metrics to Datadog",
+    "title": "Upstash",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "Upstash Database Metrics",
+        "image_url": "assets/images/upstash-dashboard.png"
+      }
+    ],
+    "classifier_tags": [
+      "Supported OS::Linux",
+      "Supported OS::Windows",
+      "Supported OS::macOS",
+      "Offering::Integration",
+      "Category::Cloud"
```

Comment:
Are there any other applicable categories? You can find the full list here: https://docs.datadoghq.com/developers/integrations/check_references/#classifier-tags